src/main/java/com/avail/interpreter/Interpreter.java

```diff
@@ -1469,13 +1571,133 @@ public Result afterAttemptPrimitive (
 		return success;
 	}
 
+	/** The method {@link #afterAttemptPrimitive(Primitive, long, Result)}. */
+	public static CheckedMethod afterAttemptPrimitiveMethod = instanceMethod(
+		Interpreter.class,
+		"afterAttemptPrimitive",
+		Result.class,
+		Primitive.class,
+		long.class,
+		Result.class);
+
 	/**
-	 * The (bottom) portion of the call stack that has been reified.  This must
-	 * always be either an {@link A_Continuation} or {@code null}, but it's
-	 * typed as {@link AvailObject} to avoid potential JVM runtime checks.
+	 * The (bottom) portion of the call stack that has been reified. This must
+	 * always be either an {@link A_Continuation}, {@link NilDescriptor#nil}, or
+	 * {@code null}, but it's typed as {@link AvailObject} to avoid potential
+	 * JVM runtime checks.
+	 */
+	private @Nullable AvailObject reifiedContinuation = null;
+
+	/**
+	 * Answer the (bottom) portion of the call stack that has been reified.
+	 * This must always be either an {@link A_Continuation},
+	 * {@link NilDescriptor#nil}, or {@code null}, but it's typed as
```

Comment:
An empty call stack is nil, but null indicates that there is no current fiber.  The latter case fails nicely instead of being mistaken for the former, which would be *very* hard to debug.  Fibers abruptly returning intermediate results.  *shudder*